### DISCUSSION TITLE
Graph Level attributes

### DISCUSSION THREAD

**Discussion by vlatorre847** [pyg-team/pytorch_geometric#7636] [Q&A]
My problem consists in obtaining a real number given a graph in input. All the nodes and the arcs of the graph have their own attributes.

Furthermore, I have an analytical method that also performs the same task, and I was wondering if I can give the output of such method to the GNN in the layer of the graph level classifier.

Regards,

V. L.

**Comment by rusty1s** [comment]
So you are dealing with graph-level regression? In this case, the general pipeline looks as
```
GNN -> Graph Readout -> MLP -> MSE loss
```
[Here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pna.py) is an example of that.
